entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -145,11 +152,46 @@ class EntityOperationHandler(
         return ResponseEntity.status(HttpStatus.OK).body(batchOperationResult)
     }
 
-    private fun extractAndParseBatchOfEntities(payload: String): Pair<List<Map<String, Any>>, List<NgsiLdEntity>> {
-        val extractedEntities = JsonUtils.parseListOfEntities(payload)
-        return Pair(
-            extractedEntities,
-            JsonLdUtils.expandJsonLdEntities(extractedEntities).map { it.toNgsiLdEntity() }
-        )
+    private fun extractAndParseBatchOfEntities(payload: String) =
```

Comment:
When it is not obvious (and it is not here), add explicitely the return type in the function signature